examples/fluids/shallow-water/shallowwater.c

```diff
@@ -378,22 +378,17 @@ int main(int argc, char **argv) {
                             &T);
   CHKERRQ(ierr);
 
-  // Pass transformation matrix to context for setup operator
-  problCtxData.T = T;
-
-  ierr = DMGetCoordinatesLocal(dm, &Xloc); CHKERRQ(ierr);
-  ierr = VecDuplicate(Xloc, &Xpanelsloc); CHKERRQ(ierr);
   // Transform coordinate according to their panel systems
-  ierr = TransformCoords(dm, Xloc, ncompx, edgenodes, nedgenodes, &physCtxData,
-                         &Xpanelsloc); CHKERRQ(ierr);
+//  ierr = TransformCoords(dm, Xloc, ncompx, edgenodes, nedgenodes, &physCtxData,
+//                         &Xpanelsloc); CHKERRQ(ierr);
```

Comment:
@jedbrown I commented out the call to this function because I think after our conversation this is the part that doesn't make sense anymore in the code. 
For testing I simply use the same as the `TESTARGS` for now, with some extra `-dm_view` options, i.e.,
```
./shallowwater -dm_refine 1 snes_fd_color -ts_fd_color -initref_dm_view -dm_view -degree 1 -qextra 0
```

Reply:
@jedbrown I don't know if you saw this message. Just in case...